package.json

```diff
@@ -20,20 +20,28 @@
     "update-ts-references": "update-ts-references"
   },
   "devDependencies": {
+    "@babel/core": "^7.12.3",
+    "@octokit/core": "^3.2.1",
```

Comment:
octokit is a peer dep of a dependency of Lerna. Adding it here silences an unmet peer dep warning.